conn.go

```diff
@@ -126,6 +134,37 @@ func DialTLS(network, addr string, config *tls.Config) (*Conn, error) {
 	return conn, nil
 }
 
+// DialURL connects to the given ldap URL vie TCP using tls.Dial or net.Dial if ldaps://
+// or ldap:// specified as protocol. On success a new Conn for the connection
+// is returned.
+func DialURL(addr string) (*Conn, error) {
```

Comment:
can we have an *tls.Config as second parameter which is used when non-nil for TLS connections?

Reply:
Someone why wants full control should use DialTLS. This function is intended to be a short-cut for common usage.